packages/@react-spectrum/overlays/src/Popover.tsx

```diff
@@ -35,7 +35,8 @@ interface PopoverProps extends Omit<AriaPopoverProps, 'popoverRef' | 'maxHeight'
   onExit?: () => void,
   container?: HTMLElement,
   disableFocusManagement?: boolean,
-  enableBothDismissButtons?: boolean
+  enableBothDismissButtons?: boolean,
+  onDismiss?: () => void
```

Comment:
Another change to consider, required to make sure focus is restored to the parent menu when a submenu popover is closed via the visually hidden dismiss button since we have disabled our overlay FocusScope management. At the moment this change applies this only to RSP Popover but we should consider adding it to aria Popover. Only thing that gave me pause was that the hidden dismiss buttons aren't really a part of the hooks persay (no concept of dismissButtonProps in usePopover since they are up to the user's discretion to add) but they are provided as a part of RAC Popover.

If we decide that we want to include it in `usePopover` I'll add that when I finish the aria example for submenu in https://github.com/adobe/react-spectrum/pull/5415/files